app/views/layouts/application.html.slim

```diff
@@ -44,3 +44,6 @@ html
         'Created with
         i.fa.fa-heart
         'by the SUSE team
+      p
```

Comment:
Could you include the `p` tag inside of the if statement. Otherwise for non-admin users they will get an extra empty paragraph :)